MODULE.bazel

```diff
@@ -6,8 +6,8 @@ module(
     version = "0.0.0",
 )
 
-bazel_dep(name = "aspect_bazel_lib", version = "1.30.2")
-bazel_dep(name = "aspect_rules_js", version = "1.26.0")
+bazel_dep(name = "aspect_bazel_lib", version = "1.32.1")
```

Comment:
Some interaction between the aspect_bazel_lib in root WORKSPACE.bazel and the version here leads to this error:
```
ERROR: /home/runner/.cache/bazel/_bazel_runner/75700d67e261d1f3b82aa7f70f76b4b9/external/expand_template_toolchains/BUILD.bazel:19:10: no such target '@aspect_bazel_lib~1.31.2//lib:expand_template_toolchain_type': target 'expand_template_toolchain_type' not declared in package 'lib' defined by /home/runner/.cache/bazel/_bazel_runner/75700d67e261d1f3b82aa7f70f76b4b9/external/aspect_bazel_lib~1.31.2/lib/BUILD.bazel (Tip: use `query "@aspect_bazel_lib//lib:*"` to see all the targets in that package) and referenced by '@expand_template_toolchains//:darwin_arm64_toolchain'
ERROR: /home/runner/.cache/bazel/_bazel_runner/75700d67e261d1f3b82aa7f70f76b4b9/external/bazel_tools/src/tools/launcher/BUILD:9:14: While resolving toolchains for target @bazel_tools//src/tools/launcher:launcher: invalid registered toolchain '@expand_template_toolchains//:darwin_arm64_toolchain': Analysis failed
```
if we also don't bump the version here.

Erring on the side of bumping this version so the expand_template toolchain is available for users.